packages/create-svelte/shared/README.md

```diff
@@ -35,6 +35,7 @@ By default, `npm run build` will generate a Node app that you can run with `node
 
 - [@sveltejs/adapter-node](https://github.com/sveltejs/kit/tree/master/packages/adapter-node)
 - [@sveltejs/adapter-static](https://github.com/sveltejs/kit/tree/master/packages/adapter-static)
+- [@sveltejs/adapter-cloudflare-workers](https://github.com/sveltejs/kit/tree/master/packages/adapter-cloudflare-workers)
```

Comment:
Should we add this to https://github.com/sveltejs/kit/blob/master/documentation/docs/10-adapters.md instead or in addition? We could point this readme to that file

Reply:
yep, have updated the README and opened https://github.com/sveltejs/kit/pull/1056